app/views/posts/show.html.erb

```diff
@@ -1,10 +1,17 @@
 <main id="content">
   <br>
- <%= link_to @post.title, @post.url %>
+  <%= link_to @post.title, @post.url %>
   <br>
-    Posted by <%= link_to @post.user.name, @post.user %>
-    <%= time_ago_in_words(@post.created_at) %> ago.
+  Posted by <%= link_to @post.user.name, @post.user %>
+  <%= time_ago_in_words(@post.created_at) %> ago.
   <br>
+  <%= pluralize(@post.upvote_count, "upvote") %>
+  <%= pluralize(@post.downvote_count, "downvote") %>
+  <br>
+  <%= render partial: 'vote/button',
+             locals: { votable: @post, value: "+1", dir: post_path(@post.id) } %>
+  <%= render partial: 'vote/button',
+             locals: { votable: @post, value: "-1", dir: post_path(@post.id) } %>
```

Comment:
As for the `dir` parameter. I wonder if there's a better way. You can do `redirect_to :back` if that is applicable to this use case so you don't have to pass in the `redir_path` parameter.